src/components/header.jsx

```diff
@@ -0,0 +1,48 @@
+import PropTypes from "prop-types"
+import React from "react"
+import { AppBar, Link, Typography, Toolbar, Box } from "@material-ui/core"
+
+import LocaleSelector from "./LocaleSelector"
+import { LOCALES } from '../const'
+import enFlag from "../images/flags/uk.svg";
+import ruFlag from "../images/flags/russia.svg";
+import byFlag from "../images/flags/belarus.svg";
+
+const { en, ru, by } = LOCALES
+const locales = [
+  {
+    name: en,
+    icon: enFlag,
+    url: '/'
+  },
+  {
+    name: ru,
+    icon: ruFlag,
+    url: `/${ru}`
+  },
+  {
+    name: by,
+    icon: byFlag,
+    url: `/${by}`
+  }
+]
+
+const Header = ({ siteTitle = '' }) => (
```

Comment:
May bee you should use 
```
Header.defaultProps = {
siteTitle = '',
}
```
 instead of setting it in the parameter siteTitle  = ''

[Airbnb React/JSX Style Guide](https://airbnb.io/javascript/react/)

Reply:
@pnhepw Just shorter)
Done